.github/workflows/release_linux_x86_64.yml

```diff
@@ -34,11 +34,24 @@ jobs:
         python-version: ${{ matrix.python-version }}
         architecture: ${{ matrix.architecture }}
 
+    - name: Install protobuf
```

Comment:
why are you installing protobuf here?
entrypoint.sh does this right?
https://github.com/onnx/onnx/blob/master/.github/workflows/manylinux/entrypoint.sh#L18

Reply:
entrypoint.sh installs protobuf inside the docker for building the wheel. The protobuf installation here is in the GitHub Action environment for testing the wheel.

Reply:
can you add this comment to the yml to make this more clear. Thanks!

Reply:
also since this is relevant for testing can we move this below just before testing starts?

Reply:
Updated. Thanks for the suggestions!